x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/plan/logical/command/sys/SysTypesTests.java

```diff
@@ -32,80 +37,85 @@
 
     private final SqlParser parser = new SqlParser();
 
-    private Tuple<Command, SqlSession> sql(String sql) {
+    private Tuple<Command, SqlSession> sql(String sql, Mode mode, SqlVersion version) {
+        SqlConfiguration configuration = new SqlConfiguration(DateUtils.UTC, Protocol.FETCH_SIZE,
+            Protocol.REQUEST_TIMEOUT, Protocol.PAGE_TIMEOUT, null, mode, null, version, null, null, false, false);
         EsIndex test = new EsIndex("test", SqlTypesTests.loadMapping("mapping-multi-field-with-nested.json", true));
-        Analyzer analyzer = new Analyzer(SqlTestUtils.TEST_CFG, new FunctionRegistry(), IndexResolution.valid(test), null);
+        Analyzer analyzer = new Analyzer(configuration, new FunctionRegistry(), IndexResolution.valid(test), null);
         Command cmd = (Command) analyzer.analyze(parser.createStatement(sql), false);
 
         IndexResolver resolver = mock(IndexResolver.class);
-        SqlSession session = new SqlSession(SqlTestUtils.TEST_CFG, null, null, resolver, null, null, null, null, null);
+        SqlSession session = new SqlSession(configuration, null, null, resolver, null, null, null, null, null);
         return new Tuple<>(cmd, session);
     }
+    private Tuple<Command, SqlSession> sql(String sql) {
+        return sql(sql, randomFrom(Mode.values()), randomBoolean() ? null : SqlVersion.fromId(Version.CURRENT.id));
+    }
 
-    public void testSysTypes() {
-        Command cmd = sql("SYS TYPES").v1();
+    static final List<String> ALL_TYPES = asList("BYTE", "LONG", "BINARY", "NULL", "INTEGER", "SHORT", "HALF_FLOAT",
```

Comment:
Why was this extracted to top level - I only see it used inside its original method.

Reply:
No actual reason, reverting.

Reply:
(This was probably also the UL PR, where it was re-used. Sorry.)

Reply:
No worries @bpintea, please! It's my fault that I included all the changes without any filtering.